topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -385,7 +385,7 @@ The BAM file contains information for all our reads, making it difficult to insp
 >    >
 >    {: .comment}
 >
-> 2. **IGV** {% icon tool %}: Zoom to `chr4:540,000-560,000` (Chromosome 4 between 540 kb to 560 kb)
+> 2. [**IGV**](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_bedtoigv/2.30.0) {% icon tool %}: Zoom to `chr4:540,000-560,000` (Chromosome 4 between 540 kb to 560 kb)
```

Comment:
There is no IGV link as it is outside Galaxy

Reply:
That was one of my questions to you for our coming meeting ;), great so I will remove the tool link